rover_camera/src/glowstick_detection/glowstick_detector.hpp

```diff
@@ -0,0 +1,33 @@
+#ifndef GLOWSTICK_DETECTOR_HPP
+#define GLOWSTICK_DETECTOR_HPP
+
+#include "glowstick.hpp"
+#include "glowstick_position_estimator.hpp"
+
+class GlowstickDetector
+{
+    enum
+    {
+        WHITE,
+        BLUE,
+        RED
+    };
```

Comment:
```suggestion
    enum class
    {
        WHITE = 0,
        BLUE = 1,
        RED = 2,
        eLAST
    };
```

eLAST is often used for loops to avoid magic numbers